src/power_grid_model_io/data_stores/excel_file_store.py

```diff
@@ -174,7 +174,8 @@ def _handle_duplicate_columns(self, data: pd.DataFrame, sheet_name: str) -> pd.D
                     new_name=new_name,
                     col_idx=col_idx,
                 )
-                columns[col_idx] = new_name
+                new_name = new_name[0] if isinstance(new_name, tuple) else new_name
+                columns[col_idx] = (new_name, columns[col_idx][1])
```

Comment:
```
new_name = new_name[0] if isinstance(new_name, tuple) else new_name
full_new_name = (new_name, columns[col_idx][1])
# log warning with full_new_name
columns[col_idx] = full_new_name
```

Reply:
```python
# log warning with full_new_name
```
The warning is issued later in the `change name` part

Reply:
Oh. Earlier, the tuple was being used in the warning above this sign, right? (Edited the previous comment)

Reply:
Right

Reply:
This and below, will be further processed.